tests/integration/cht-form/default/person-edit.wdio-spec.js

```diff
@@ -40,9 +40,6 @@ describe('cht-form web component - Edit Person Form', () => {
             created_by_place_uuid: ''
           }
         },
-        meta: {
```

Comment:
I do not think this block should have been included here in the first place. It was causing a warning to be printed to the browser logs.